### PR TITLE
OP-1035 - Patient consent

### PR DESCRIPTION
**Pull request by goto-eof**
See OP-1035.

- [x] Add paragraph with descriptions
- [x] Description for Patient Consensus flag OP-1036
- [x] Description for Additional Service flag OP-1038
- [ ] Update screenshots

### CODE REVIEW

File: doc_user/UserManual.adoc [hunk 1]
```diff
@@ -1584,6 +1584,13 @@ To modify the anamnesis data of a patient, select the patient in the *_Patient B
 
 image:image247.png[Anamnesis]
 
+====== 8.4.7 Patient consensus
+Consent from a patient is needed regardless of the procedure, whether it's a physical examination or something else. The principle of consent is an important part of medical ethics and international human rights law.
+ In Open Hospital there are three types of Patient consent:
+- Consent to treatment: consent to treatment means a person must give permission before they receive any type of medical treatment, test or examination. This field is mandatory.
+- Administrative consent: administrative consent order means a legal agreement signed by the Director and the owner or operator of a Poultry Feeding Operation.
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]: Is this really correct?    `Poultry Feeding Operation`

File: doc_user/UserManual.adoc [hunk 2]
```diff
@@ -1584,6 +1584,13 @@ To modify the anamnesis data of a patient, select the patient in the *_Patient B
 
 image:image247.png[Anamnesis]
 
+====== 8.4.7 Patient consensus
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]:
```suggestion
====== 8.4.7 Patient Consensus
```

File: doc_user/UserManual.adoc [hunk 3]
```diff
@@ -1584,6 +1584,13 @@ To modify the anamnesis data of a patient, select the patient in the *_Patient B
 
 image:image247.png[Anamnesis]
 
+====== 8.4.7 Patient Consensus
+Consent from a patient is needed regardless of the procedure, whether it's a physical examination or something else. The principle of consent is an important part of medical ethics and international human rights law.
+ In Open Hospital there are three types of Patient consent:
+- Consent to treatment: consent to treatment means a person must give permission before they receive any type of medical treatment, test or examination. This field is mandatory.
+- Administrative consent
+- Consent to additional services
```
Comment (mwithi) [inline doc_user/UserManual.adoc]:
```suggestion
- Consent to additional services: this consent is for the opt-in and opt-out from additional services the hospital provides (e.g. special mailing, contacts, etc...)
- Administrative flag: this flag is important if the hospital wants to flag the patient due to other administrative reason (e.g. undue payments, technical issues, etc...)
```

**Comment by mwithi** [comment]
Found that "Administrative Consent" is not easy to explain and maybe we should discuss it a little bit more

**Comment by mwithi** [comment]
Superseded by https://github.com/informatici/openhospital-doc/pull/226

**Pull request closed without merging**